cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/TimeSeriesGroupEndpointInput.java

```diff
@@ -56,8 +60,8 @@ public static Post post(TimeSeriesGroup timeSeriesGroup) {
         return new Post(timeSeriesGroup);
     }
 
-    public static Patch patch(String originalGroupId, TimeSeriesGroup timeSeriesGroup) {
-        return new Patch(originalGroupId, timeSeriesGroup);
+    public static Patch patch(String office, String originalGroupId, TimeSeriesGroup timeSeriesGroup) {
```

Comment:
What office is this?

Reply:
Updated to reflect that this is the user operating office

Reply:
there is no user office in this context

Reply:
swapped to groupOffice